src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+abstract class Figure implements Area, Drawable {
```

Comment:
Don't put all behavior into a single interface if the methods are conceptually different from each other. It seems that 'Area' and 'Drawable' interfaces may contain different concepts and should not be implemented by the same class if they don't share the same conceptual purpose.